.gitmodules

```diff
@@ -1,6 +1,3 @@
-[submodule "Vendor/sqlcipher"]
```

Comment:
Carthage wasn't building the amalgamation target properly, if I recall correctly. The easiest solution was to remove sqlcipher as a submodule and manually add some files that Xcode complained were missing. Now with Xcode 7.3, though, it doesn't build at all. I haven't been able to get it working.